### PR TITLE
[14.0][ADD] product_packaging_archive

### PR DESCRIPTION
**Pull request by mt-software-de**
Add an active field on product packaging
 cc @jbaudoux

### CODE REVIEW

File: product_packaging_archive/views/product_packaging_views.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="product_packaing_form_view">
```
Comment (jbaudoux) [inline product_packaging_archive/views/product_packaging_views.xml:6]:
```suggestion
    <record model="ir.ui.view" id="product_packaging_form_view">
```

File: product_packaging_archive/views/product_packaging_views.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        <field name="arch" type="xml">
+            <field name="company_id" position="after">
+                <field name="active" />
+            </field>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window" id="product.action_packaging_view">
+        <field name="context">{'active_test': False}</field>
+    </record>
```
Comment (jbaudoux) [inline product_packaging_archive/views/product_packaging_views.xml:36]: This should not be here but on the product form view to ensure we can activate/inactivate a product packaging.

**Review by jbaudoux — CHANGES_REQUESTED** [review]
It's also missing the archive flag on the product packaging form view on the product view

**Comment by mt-software-de** [comment]
After @jbaudoux argument convinced me, I close this PR and open a new for delivery packaging. 
https://github.com/OCA/delivery-carrier/pull/680

**Pull request closed without merging**